internal/auth0/resourceserver/resource.go

```diff
@@ -103,14 +103,19 @@ func NewResource() *schema.Resource {
 			"token_dialect": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
Is this because we provide a default `token_dialect` value, when it is not provided in configuration?